### ISSUE TITLE
Hello, linux noobie here, does anyone know what causes this problem and how to fix this?

### ISSUE THREAD

**Issue by max234234324324** [dnschneid/crouton#5067]
[
![B5F7B4AF-F18B-4FCB-9C35-C34F4B21E97D](https://github.com/dnschneid/crouton/assets/167622033/bc5c6ccc-3d7f-45f6-a99a-53c82655a99a)
](url)

The culprit seems to be in something called the register core dump handler, chroot and xenial though I dont know what that really is.

**Comment by CroutonIsFun** [comment]
The situation is difficult for newcomers. The original instructions no longer work. All I can tell you is to browse the discussions tab. Crouton still works, but requires some special "hacks". I've already written several "how-to's", which you'll find in the discussion area.

**Comment by max234234324324** [comment]
Ah thx, I found this useful instruction from you.

https://github.com/dnschneid/crouton/discussions/5065

Should I use this/ is it still up to date and if yes, could you please explain what you exactly mean with this or how I could achieve this via which commands:

 _"such as setting up sshd inside the chroot, then ssh into your chroot from the crosh shell."_

**Comment by CroutonIsFun** [comment]
@max234234324324 

The text you wish to read is at [#5013](https://github.com/dnschneid/crouton/discussions/5013)

**Comment by max234234324324** [comment]
ah THX ill try👍

**Comment by max234234324324** [comment]
Ive attempted your guide but unfortunately while entering sudo enter-chroot it displays these messages about something not working right.
![image](https://github.com/dnschneid/crouton/assets/167622033/a7d817ec-4503-4b58-a909-35f7acc6728c)
![image](https://github.com/dnschneid/crouton/assets/167622033/8b0e0311-b7ef-4c7a-9c83-3979176f5f76)

**Comment by CroutonIsFun** [comment]
@max234234324324

You need to use the silence branch, as the crash (Chrome Audio System) files no longer work:

`sudo CROUTON_BRANCH=silence crouton -r bullseye -t xfce` 

Or similar

**Comment by nethershaw** [comment]
> You need to

Halt. Point of order.

Why is this buried in a discussion thread instead of somewhere obvious, requiring now that anyone interested is forced to recurse through this and other issues to find _you_ mentioning something potentially useful?

**Comment by max234234324324** [comment]
You said I had to exit chroot which I tried to find how to. I thought I found it on another forum but turns out I did the wrong thing so I want to also know how to exit chroot and also after the chroot exit fail I tried to redo the process by powerwashing my chromebook and entering sudo CROUTON_BRANCH=silence crouton -r bullseye -t xfce, doesnt work anymore, while first it did.
![image](https://github.com/dnschneid/crouton/assets/167622033/0b796d6c-878f-4cd5-a4a2-4133b4c0b738)

**Comment by CroutonIsFun** [comment]
> > You need to
> 
> Halt. Point of order.
> 
> Why is this buried in a discussion thread instead of somewhere obvious, requiring now that anyone interested is forced to recurse through this and other issues to find _you_ mentioning something potentially useful?

Yeah, I agree. The maintainer has decided to keep the original instructions as a sort of repository. I've written a whole new "how to" on how to get back to some sort of reasonableness.


https://github.com/dnschneid/crouton/wiki/(CIF000)-A-completely-%22new%22-crouton:-follow-these-instructions

**Comment by Qwer-TeX** [comment]
@CroutonIsFun 
The links doesn't display the new instructions but instead the title.

![Selection_001](https://github.com/dnschneid/crouton/assets/128016252/78b11100-0e39-41a9-85ee-71623c6a811e)
